tests/regression/test_scaled_mass.py

```diff
@@ -30,27 +30,23 @@ def mesh():
                          ['scalar', 'vector', 'tensor'])
 def test_math_functions(mesh, expr, value, typ, fs_type):
     if typ == 'Function':
-        if fs_type == 'vector':
-            V = VectorFunctionSpace(mesh, 'CG', 1)
-        elif fs_type == 'tensor':
-            V = TensorFunctionSpace(mesh, 'CG', 1)
-        else:
-            V = FunctionSpace(mesh, 'CG', 1)
-        f = Function(V)
-        f.assign(value)
-        if fs_type == 'vector':
-            f = dot(f, f)
-        elif fs_type == 'tensor':
-            f = inner(f, f)
+        family, degree = 'CG', 1
     elif typ == 'Constant':
-        if fs_type == 'vector':
-            f = Constant([value, value], domain=mesh)
```

Comment:
Is this really the right thing to do in the test? Can't we just get rid of `domain=mesh`? Is there a good reason to get rid of `Constant` tests now we also have real functions?

Reply:
I agree